src/assets/lang_turkish.svg

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<svg xmlns="http://www.w3.org/2000/svg" width="1200" height="800">
```

Comment:
please remove `width` and `height` and add `viewBox="0 0 1200 800"` in this tag. This also fixes the ![resize bug](https://github.com/ProjektKuro/kuro-web/pull/46#discussion_r403687530)